beacon_node/beacon_chain/src/light_client_finality_update_verification.rs

```diff
@@ -67,7 +67,8 @@ impl<T: BeaconChainTypes> VerifiedLightClientFinalityUpdate<T> {
         chain: &BeaconChain<T>,
         seen_timestamp: Duration,
     ) -> Result<Self, Error> {
-        let gossiped_finality_slot = light_client_finality_update.finalized_header.beacon.slot;
```

Comment:
Note that having to compute the full LightClientFinality update every time verification is called is not the optimal final strategy. PR https://github.com/sigp/lighthouse/pull/4969 already switches to computing this updates eagerly in another thread and using the cached values here on verification.

So I'm not sure if it's just better to merge https://github.com/sigp/lighthouse/pull/4969 first so we don't have to deal with the runtime here.

Reply:
Yeah I also think it makes sense to wait to merge #4969 first